env/dev/env/ios/main.cljs

```diff
@@ -3,20 +3,27 @@
             [re-frisk-remote.core :as rr]
             [status-im.ios.core :as core]
             [status-im.utils.handlers :as utils.handlers]
-            [figwheel.client :as figwheel :include-macros true]
+            [figwheel.client :as fw]
+            [env.config :as conf]
+            [env.util]
             [cljs.pprint]))
 
 (enable-console-print!)
 
+(assert (exists? core/init) "Fatal Error - Your core.cljs file doesn't define an 'init' function!!! - Perhaps there was a compilation failure?")
+(assert (exists? core/app-root) "Fatal Error - Your core.cljs file doesn't define an 'app-root' function!!! - Perhaps there was a compilation failure?")
+
 (def cnt (r/atom 0))
 (defn reloader [] @cnt [core/app-root])
 (def root-el (r/as-element [reloader]))
 
-(figwheel/watch-and-reload
-  :websocket-url "ws://localhost:3449/figwheel-ws"
-  :heads-up-display false
-  :jsload-callback #(swap! cnt inc))
+(fw/start {
```

Comment:
Why the change to `start` ?

Reply:
Same reason as above.
Will fix

Reply:
It would be interesting to know what the difference is between both.

Reply:
according to here
https://github.com/bhauman/lein-figwheel/blob/master/support/src/figwheel/client.cljs#L479

`watch-and-reload` is a legacy interface which calls `start`

Reply:
Let's keep `start` then!